compiler/fir/resolve/src/org/jetbrains/kotlin/fir/resolve/transformers/IntegerLiteralTypeApproximationTransformer.kt

```diff
@@ -77,13 +77,28 @@ class IntegerLiteralTypeApproximationTransformer(
         // e.g. Byte doesn't have `and` in member scope. It's an extension
         if (resultSymbol == null) return functionCall.compose()
```

Comment:
I found that the original `calleeReference.resolvedSymbol` (after resolution) is of `FirIntegerOperator` from `FirJAvaModuleBasedSession`, while this, kind of reloaded, result symbol is `FirSimpleFunctionImpl` from `FirLibrarySession`. I'm not sure 1) why the latter was not found as a resolution candidate; and 2) why the former was not reloaded again from the member scope lookup.